packages/core/test/store.spec.ts

```diff
@@ -148,6 +148,43 @@ describe("Store", () => {
       await store.delete("key", { code: 0, message: "reason" });
       expect(store.length).to.equal(0);
     });
+    it("should add deleted key to the recentlyDeleted list", async () => {
+      const key = "key";
+      const value = "value";
+      await store.set(key, value);
+      await store.delete(key, { code: 0, message: "reason" });
+      try {
+        await store.get(key);
+      } catch (e) {
+        expect(e.message).to.equal(
+          `Missing or invalid. Record was recently deleted - mock-entity: ${key}`,
+        );
+      }
+    });
+    it("should cleanup recentlyDeleted when size limit is reached", async () => {
+      //@ts-ignore
```

Comment:
Why so many `@ts-ignore` needed here?

Reply:
Updated to `//@ts-expect-error`. Reason is that its not public property but we're still acceesing it for tests